content/kotlin/concepts/datetime/terms/DayOfWeek/DayOfWeek.md

```diff
@@ -0,0 +1,26 @@
+---
+Title: 'DayOfWeek'
+Description: 'The DayOfWeek method will return the '
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Method'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **DayOfWeek** method is used to return the DayOfWeek instance for the week day number, starting with 1 as Monday. 
+## Syntax
+'''pseudo
+fun DayOfWeek(isoDayNumber: Int): DayOfWeek
+'''
+The integer value input into the method must correspond to one of the seven days in the DayOfWeek enumeration class. The numbering follows the ISO-8601 week day number, where Monday is 1 and Sunday is 7. 
+
+The enumeration class enteries have two properties: Name, which is represented by a string, and the ordinal, which is represented as an integer. 
+## Example
+'''kolin
+println(DayOfWeek(3))
```

Comment:
```suggestion
import java.time.DayOfWeek

fun main() {
    val day = DayOfWeek.of(3)
    println(day)
}
```
Made some changes in this code snippet so it can work if someone tries to compile it.